Python/Product/Uwp.Interpreter/PythonUwpInterpreterFactoryProvider.cs

```diff
@@ -21,106 +21,381 @@
 using System.Linq;
 using System.Reflection;
 using Microsoft.PythonTools.Interpreter;
+using Microsoft.PythonTools.Infrastructure;
```

Comment:
That's the main class we want to avoid having people depend on :) . You're already using Path functions throughout, so switch to those.